fenix/app/src/main/res/values/preference_keys.xml

```diff
@@ -25,7 +25,7 @@
     <string name="pref_key_delete_browsing_data_on_quit" translatable="false">pref_key_delete_browsing_data_on_quit</string>
     <string name="pref_key_delete_open_tabs_on_quit" translatable="false">pref_key_delete_open_tabs_on_quit</string>
     <string name="pref_key_delete_browsing_history_on_quit" translatable="false">pref_key_delete_browsing_history_on_quit</string>
-    <string name="pref_key_delete_cookies_on_quit" translatable="false">pref_key_delete_cookies_on_quit</string>
+    <string name="pref_key_delete_cookies_and_site_data_on_quit" translatable="false">pref_key_delete_cookies_and_site_data_on_quit</string>
```

Comment:
The below comment about strings is not applicable here

Reply:
I'm not sure what you mean by this. Does this mean I have nothing to change here ?

Reply:
Yeah, that's right!